File: atividade_6/nomeS.py
nome_completo = input("Digite o nome completo: ")


palavras = nome_completo.split()

if len(palavras) >= 2:
    sobrenome = palavras[-1]
    nome = ' '.join(palavras[:-1])  
    print(f"{sobrenome.upper()}/{nome}")
else:
    print("Nome incompleto ou inválido.")

File: atividade_6/nomeS.c
#include <stdio.h>
#include <string.h>

int main() {
    char nome[100];
    printf("Digite o nome completo: ");
    fgets(nome, sizeof(nome), stdin);

    int len = strlen(nome);

    int i;
    for (i = len - 1; i >= 0; i--) {
        if (nome[i] == ' ') {
            break;
        }
    }

    if (i >= 0) {
        printf("%s/%s", &nome[i + 1], nome);
        nome[i] = '\0';  
    } else {
        printf("Nome incompleto ou inválido.\n");
    }

    return 0;
}
